rotkehlchen/api/v1/resources.py

```diff
@@ -594,26 +595,27 @@ def get(
 class ManuallyTrackedBalancesResource(BaseResource):
 
     get_schema = AsyncQueryArgumentSchema()
-    edit_schema = ManuallyTrackedBalancesSchema()
+    new_items_schema = ManuallyTrackedBalancesAddSchema()
+    edit_items_schema = ManuallyTrackedBalancesEditSchema()
```

Comment:
nitpick: As you can see from the rest of the file we use the name of the verb everywhere for the schemas
```suggestion
    put_schema = ManuallyTrackedBalancesAddSchema()
    patch_schema = ManuallyTrackedBalancesEditSchema()
```